stable/distributed-tensorflow/templates/statefulset-ps.yaml

```diff
@@ -80,7 +94,10 @@ spec:
         volumeMounts:
 {{ toYaml .Values.volumeMounts | indent 8 }}
 {{- end }}
-{{- if .Values.ps.resources }}
+{{- if gt (int .Values.worker.gpuCount) 0 }}
         resources:
```

Comment:
This change only applies container limits when using GPU.
Please restore `.Values.ps.resources` which helps operators to precisely define container limits

Reply:
Done